metis-authentication/metis-authentication-rest/src/main/resources/create_tables.sql

```diff
@@ -0,0 +1,28 @@
+CREATE TABLE IF NOT EXISTS metis_users (
+ user_id VARCHAR(100),
+ email VARCHAR(40) PRIMARY KEY,
+ last_name VARCHAR(40),
+ first_name VARCHAR(40),
+ password VARCHAR(255),
+ organization_id VARCHAR(100),
+ organization_name VARCHAR(100),
+ account_role VARCHAR(40) CHECK (account_role IN ('METIS_ADMIN', 'EUROPEANA_DATA_OFFICER', 'PROVIDER_VIEWER') ),
```

Comment:
You can't make this an enum value?

Reply:
I could. There is an issue with ENUM types though..
I cannot say IF NOT EXISTS. I can only DROP it first, and if I do that it complains that it's used in other tables..
Do you know perhaps a better way to do this?

Reply:
Or maybe I'll remove the IF NOT EXISTS on everything?
And it will always fails if it is executed on a db that already has any of the tables?

Reply:
Maybe not overcomplicate things... In fact, perhaps it is best to just remove the check for values? It is a text field, who cares what values are in it? The java code will handle it.

Reply:
Ok let's remove the check completely.. I've added a comment also to give a bit of info.